Dockerfile

```diff
@@ -1,11 +1,16 @@
-from ruby:2.5
+FROM ruby:2.5
 
-workdir /app
+WORKDIR /app
 
-run apt-get update && apt-get install -y qt5-default libqt5webkit5-dev \
+RUN apt-get update && apt-get install -y qt5-default libqt5webkit5-dev \
       gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x libopenscap-dev \
       postgresql-client
 
-add Gemfile Gemfile.lock ./
+COPY Gemfile* ./
+COPY entrypoint.sh ./
+RUN chmod a+x ./entrypoint.sh
```

Comment:
We can remove this line, I think. The entrypoint.sh should be copied with its permissions unchanged, and it's `a+x` in the repo.